### ISSUE TITLE
E2ETest failure

### ISSUE THREAD

**Issue by igarashitm** [atlasmap/atlasmap#2925]
> no such element: Unable to locate element: {"method":"xpath","selector":"//article[@aria-label='io.atlasmap.service.my.MyFieldActionsModel']"}
  (Session info: headless chrome=91.0.4472.77)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'p1', ip: '192.168.12.121', os.name: 'Linux', os.arch: 'amd64', os.version: '5.12.9-200.fc33.x86_64', java.version: '11.0.11'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 91.0.4472.77, chrome: {chromedriverVersion: 90.0.4430.212 (e3cd97fc771b..., userDataDir: /tmp/.org.chromium.Chromium...}, goog:chromeOptions: {debuggerAddress: localhost:38975}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 1086ae1f00babe06eee6fb6e24b8b785
*** Element info: {Using=xpath, value=//article[@aria-label='io.atlasmap.service.my.MyFieldActionsModel']}
> ...
> at io.atlasmap.standalone.E2ETest.test(E2ETest.java:133)

**Comment by igarashitm** [comment]
It'd be a timing issue, interestingly it tends to succeed in GH action, but fail on CircleCI and my local build.

**Comment by igarashitm** [comment]
Not really, it used to be using FQDN as a Java Document label but now it's a simple name. Using simple name is correct as a default display name. cf. #2919
